Doc/reference/datamodel.rst

```diff
@@ -3356,10 +3356,15 @@ left undefined.
    is called if ``type(x).__sub__(x, y)`` returns :data:`NotImplemented` or ``type(y)``
    is a subclass of ``type(x)``. [#]_
 
-   .. index:: pair: built-in function; pow
+   Note that :meth:`__rpow__` should be defined to accept an optional third
```

Comment:
Maybe you should use `object.__rpow__(self, other, modulo=None)` signature?

Reply:
Sorry, I meant L3329-3342 above.

Reply:
I see. There are other similar cases (for example `__round__`), so I will left this for other issue. Actually, `__pow__()` and `__round__()` are never called with None, so it is not necessary that they support None.